html-forms/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <label for="email">Email</label>
+      <div>
+        <input type="email" id="email">
+      </div>
+      <label for="dropdown">Why is coding cool?</label>
```

Comment:
This form control element associated with this `label` needs to have an `id` with a matching value